### DISCUSSION TITLE
Anyone else seeing RAM errors?

### DISCUSSION THREAD

**Discussion by AjazElahi** [madMAx43v3r/chia-plotter#815] [Q&A]
I've got 8 new blade servers arrived and installed a few weeks ago. Each blade has 128GB RAM (4x 32GB DDR4)
I've been plotting with MadMax and it's amazing, works really nice! 

However, 2 of my blade servers are no longer plotting and I'm seeing a warning 'Memory degraded' and it looks like 1 out of the 4 ram sticks is causing this error. 

Does MadMax eat Ramsticks? I thought RAM was more durable. I've only made about 800 plots across 8 machines... so only like 100 plots on each machine.

**Comment by SebMoore** [comment]
> I've got 8 new blade servers arrived and installed a few weeks ago. Each blade has 128GB RAM (4x 32GB DDR4)
> I've been plotting with MadMax and it's amazing, works really nice!
> 
> However, 2 of my blade servers are no longer plotting and I'm seeing a warning 'Memory degraded' and it looks like 1 out of the 4 ram sticks is causing this error.
> 
> Does MadMax eat Ramsticks? I thought RAM was more durable. I've only made about 800 plots across 8 machines... so only like 100 plots on each machine.

DRAM modules typically have a Mean Time Before Failure (MTBF) of about 50,000 hours or a little over 6 years. Now, because a stick of ram has multiple modules, for a server stick that's under constant use, it's more like 3-5 years. If your blades are used, then it is possible that the previous owner used the for something strenuous as well, and could have worn out the RAM. But yes, the "memory degraded" error is usually correct - it has identified that one of your RAM sticks is failing - advanced ECC is keeping your entire system from crashing right now, but it's probably only a matter of time unfortunately.
If your blades are new, however, it's time for a warranty claim. You have been given faulty memory.

**Reply by AjazElahi** [reply]
The blades are used (HP BL460C) but the RAM sticks are new. Thank you for the input, I will try to reseat them and see if that solves the issue but failing that, I will have to get them RMA'd.
